extractor/src/extractors/GoModExtractor.ts

```diff
@@ -84,16 +84,20 @@ export default class GoModExtractor implements Extractor {
                         }
                     }
                     break;
-
+                
+                case "retract":
                 case "exclude":
                     i++;    // exclude on subsequent lines
                     for (; i < lines.length && lines[i] !== ")"; i++) {
                         // intentionally empty
                     }
                     break;
-
+                
+                case "//":
+                    break;
+                
                 default:
-                    throw new Error(`parse error: unsupported directive: ${directive}`);
+                    console.debug(`parse error: unsupported directive: ${directive}`);
```

Comment:
Don't use console. This project uses log4js:

https://github.com/depscloud/depscloud/blob/main/extractor/src/main.ts#L4
https://github.com/depscloud/depscloud/blob/main/extractor/src/main.ts#L22